modules/tinymce/src/core/main/ts/fmt/MatchFormat.ts

```diff
@@ -191,24 +192,23 @@ const closest = (editor: Editor, names: string[]): string | null => {
 
 const canApply = (editor: Editor, name: string) => {
   const formatList = editor.formatter.get(name);
-  let startNode, parents, i, x, selector;
   const dom = editor.dom;
 
   if (formatList) {
-    startNode = editor.selection.getStart();
-    parents = FormatUtils.getParents(dom, startNode);
+    const startNode = editor.selection.getStart();
+    const parents = FormatUtils.getParents(dom, startNode);
 
-    for (x = formatList.length - 1; x >= 0; x--) {
-      selector = (formatList[x] as SelectorFormat).selector;
+    for (let x = formatList.length - 1; x >= 0; x--) {
+      const format = formatList[x];
 
       // Format is not selector based then always return TRUE
       // Is it has a defaultBlock then it's likely it can be applied for example align on a non block element line
```

Comment:
I know you didn't change this but I just spotted this typo. Up to you if you want to change it
```suggestion
      // If it has a defaultBlock then it's likely it can be applied, for example align on a non block element line
```